### DISCUSSION TITLE
Traveler's Backpack Mod (Entity and Base)

### DISCUSSION THREAD

**Discussion by WWJTLoS** [Ladysnake/Cardinal-Components-API#182] [Q&A] [answered]
Hello Everyone,
I'm in need of help because the Traveler's Backpack od for Minecraft was saying that it would work unless I downloaded any version of Cardinal Components Base and Entity. As if they have separate .jar files? Honestly, I am quite confused and would really appreciate the help!

**Comment by UpcraftLP** [comment, marked as answer]
Hello!

They're provided to mod developers as separate modules, however the jars published to Curseforge and Modrinth contain ALL modules so just use those ^^
